src/constants/index.ts

```diff
@@ -3,4 +3,4 @@ import Stripe from 'stripe';
 export * from './stripeModuleOptions';
 export * from './stripeToken';
 
-export const apiVersion: Stripe.LatestApiVersion = '2020-03-02';
+export const apiVersion: Stripe.LatestApiVersion = '2020-08-27';
```

Comment:
The tests don't seem to pass locally for me when this change is present. Can you try to run tests using `yarn install --frozen-lockfile`?

Reply:
Reverted so the tests pass using yarn. I was using npm at time of running tests.